projects/elfutils/project.yaml

```diff
@@ -0,0 +1,17 @@
+homepage: "https://sourceware.org/elfutils/"
+language: c++
+primary_contact: "izzeem@google.com"
```

Comment:
No sorry. Each person needs their own google account.
I understand it's inconvenient, but that's all we have support for now.

Reply:
@fche `libelf` is fuzzed indirectly via the libbpf project: https://storage.googleapis.com/oss-fuzz-coverage/libbpf/reports/20211030/linux/src/libbpf/elfutils/report.html and to tell libbpf and libelf bugs apart (or point the build script to commits where bugs have already been fixed) it would be great if I could have access to elfutils bug reports. Would it be OK if I added my email address to `auto_cc`? Thanks!